packages.json

```diff
@@ -6191,5 +6191,16 @@
     "description": "String algorithms with succinct data structures",
     "license": "Apache2",
     "web": "https://unicredit.github.io/cello/"
+  },
+  {
+    "name": "notmuch",
+    "url": "https://github.com/samdmarshall/notmuch.nim",
+    "method": "git",
+    "tags": [
+      "notmuch"
```

Comment:
Please come up with tags that are not just the name of the package. For example: ``mail``, ``library``, ``wrapper``